content/en/docs/instrumentation/python/automatic/logs-example.md

```diff
@@ -0,0 +1,167 @@
+pip install opentelemetry-exporter-otlp
+```
+
+The examples that follow send instrumentation results to the console. Learn more
+about installing and configuring the
+[OpenTelemetry Distro](/docs/instrumentation/python/distro) to send telemetry to
+other destinations, like an OpenTelemetry Collector.
+
+> **Note**: To use automatic instrumentation through `opentelemetry-instrument`,
+> you must configure it via environment variables or the command line. The agent
+> creates a telemetry pipeline that cannot be modified other than through these
+> means. If you need more customization for your telemetry pipelines, then you
+> need to forego the agent and import the OpenTelemetry SDK and instrumentation
+> libraries into your code and configure them there. You may also extend
+> automatic instrumentation by importing the OpenTelemetry API. For more
+> details, see the [API reference][].
+
+## Execute
+
+This section guides you through the process of executing an automatically
+instrumented logs.
+
+Open up a new terminal window and start the OTel Collector:
+
+```sh
+docker run -it --rm -p 4317:4317 -p 4318:4318 \
+  -v $(pwd)/otel-collector-config.yaml:/etc/otelcol-config.yml \
+  --name otelcol \
+  otel/opentelemetry-collector-contrib:0.76.1 \
+  "--config=/etc/otelcol-config.yml"
+```
+
+Open up another terminal and run the Python program:
+
+```sh
+source python_logs_example/bin/activate
+
+export OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED=true
+opentelemetry-instrument \
+  --traces_exporter console,otlp \
+  --metrics_exporter console,otlp \
+  --logs_exporter console,otlp \
+  --service_name python-logs-example \
+  python $(pwd)/example.py
+```
+
+Sample output:
+
+```text
+...
+ScopeSpans #0
+ScopeSpans SchemaURL:
+InstrumentationScope __main__
+Span #0
+    Trace ID       : 389d4ac130a390d3d99036f9cd1db75e
+    Parent ID      :
+    ID             : f318281c4654edc5
+    Name           : foo
+    Kind           : Internal
+    Start time     : 2023-08-18 17:04:05.982564 +0000 UTC
+    End time       : 2023-08-18 17:04:05.982667 +0000 UTC
+    Status code    : Unset
+    Status message :
+Events:
+SpanEvent #0
+     -> Name: This is a span event
+     -> Timestamp: 2023-08-18 17:04:05.982586 +0000 UTC
+
+...
+
+ScopeLogs #0
+ScopeLogs SchemaURL:
+InstrumentationScope opentelemetry.sdk._logs._internal
+LogRecord #0
+ObservedTimestamp: 1970-01-01 00:00:00 +0000 UTC
+Timestamp: 2023-08-18 17:04:05.982605056 +0000 UTC
+SeverityText: ERROR
+SeverityNumber: Error(17)
+Body: Str(This is a log message)
+Attributes:
+     -> otelSpanID: Str(f318281c4654edc5)
+     -> otelTraceID: Str(389d4ac130a390d3d99036f9cd1db75e)
+     -> otelTraceSampled: Bool(true)
+     -> otelServiceName: Str(python-logs-example)
+Trace ID: 389d4ac130a390d3d99036f9cd1db75e
+Span ID: f318281c4654edc5
+...
+```
+
+Note how in the sample output, the Span Event and the Log both have the same
+SpanID: `f318281c4654edc5`, since they're included under the same Span.
```

Comment:
```suggestion
Note that the Span Event and the Log both have the same
SpanID (`f318281c4654edc5`). This is because they're a part of the same Span.
```